bin/katana/src/args.rs

```diff
@@ -215,6 +215,8 @@ impl KatanaArgs {
         let mut apis = vec![ApiKind::Starknet];
         // only enable `katana` API in dev mode
         if self.dev {
+            apis.push(ApiKind::Dev);
+        } else {
             apis.push(ApiKind::Katana);
         }
```

Comment:
```suggestion
        let mut apis = vec![ApiKind::Starknet, ApiKind::Katana];
        
        if self.dev {
            apis.push(ApiKind::Dev);
        }
```

We want to have `starknet` and `katana` available by default. The `--dev` flag should only affect whether `dev` api is included or not.